streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -153,6 +157,8 @@ public void openDB(ProcessorContext context) {
     }
 
     public void init(ProcessorContext context, StateStore root) {
+        log.trace("Initializing store {} {}", name, loggingEnabled ? "with logging enabled" : "");
```

Comment:
Could we move this log to `AbstractTask`, before `store.init(this.processorContext, store);`? Also we can use the same logging prefix as:

```
("task [{}] Initializing store {}", store.name());
```

And since we already log the loggingEnabled information in `Stores`, I think we do not need to log it again here.